ups/eupspkg.cfg.sh

```diff
@@ -67,4 +67,14 @@ install()
 	default_install
 
 	cp -r include "$PREFIX/"
+	if [[ $OSTYPE == darwin* ]]; then
+		curdir=`pwd`
```

Comment:
You should use $(pwd) in lieu of backticks

Reply:
Good enough.